projects/ion/src/lib/table/utilsTable.ts

```diff
@@ -56,6 +57,12 @@ export interface Column<RowType = SafeAny> {
   configTooltip?: TooltipProps;
   pipe?: PipeColumn;
   hideLongData?: boolean;
+  booleanText?: ColumnBooleanText;
+}
+
+export interface ColumnBooleanText {
+  true: string;
+  false: string;
```

Comment:
I believe that is better change this to on/off, primitives values names in attributes not is recomended.